lua/which-key/keys.lua

```diff
@@ -379,6 +379,10 @@ function M.check_health()
       end
     )
   end
+  if next(M.duplicates) == nil then
+    vim.fn["health#report_ok"]("No conflicting keymaps found")
```

Comment:
`health#report_ok` is [deprecated](https://neovim.io/doc/user/deprecated.html):
![image](https://user-images.githubusercontent.com/50843046/209436179-3784f12f-9ceb-49bd-b581-6ba441893991.png)

Would be nice, if you could change that to
```lua
vim.health.report_ok("No conflicting keymaps found")
```

Reply:
I chose the deprecated version since all reports are via deprecated functions (example below) https://github.com/folke/which-key.nvim/blob/b55dbf915a50c897500566879c140c2308244336/lua/which-key/keys.lua#L375
I agree we should be using the respective Lua function, but in that case it makes sense to update all reports, which goes beyond the scope of fixing issue https://github.com/folke/which-key.nvim/issues/368